client/jetpack-onboarding/steps/stats.jsx

```diff
@@ -0,0 +1,132 @@
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import FormattedHeader from 'components/formatted-header';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import Tile from 'components/tile-grid/tile';
+import TileGrid from 'components/tile-grid';
+import { addQueryArgs } from 'lib/route';
+import { getUnconnectedSiteUrl } from 'state/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+import { JETPACK_ONBOARDING_STEPS as STEPS } from '../constants';
+import { saveJetpackOnboardingSettings } from 'state/jetpack-onboarding/actions';
+
+class JetpackOnboardingStatsStep extends React.Component {
+	componentDidUpdate() {
+		const { action, activatedStats, isConnected } = this.props;
+
+		if ( isConnected && activatedStats === false && action === 'activate_stats' ) {
+			this.activateStats();
+		}
+	}
+
+	handleActivateStats = () => {
+		this.props.recordJpoEvent( 'calypso_jpo_stats_clicked' );
+
+		if ( ! this.props.isConnected ) {
+			return;
+		}
+
+		this.activateStats();
+	};
+
+	handleStatsNextButton = () => {
+		this.props.recordJpoEvent( 'calypso_jpo_stats_next_clicked' );
+	};
+
+	activateStats() {
+		this.props.saveJetpackOnboardingSettings( this.props.siteId, {
+			stats: true,
+		} );
+	}
+
+	renderSuccess() {
+		const { getForwardUrl, translate } = this.props;
+		const headerText = translate( 'Success! Jetpack is now collecting valuable stats.' );
+
+		return (
+			<Fragment>
+				<FormattedHeader headerText={ headerText } />
+
+				<TileGrid>
+					<Tile
+						buttonLabel={ translate( 'Continue' ) }
+						image={ '/calypso/images/illustrations/type-business.svg' }
+						onClick={ this.handleStatsNextButton }
+						href={ getForwardUrl() }
+					/>
+				</TileGrid>
+			</Fragment>
+		);
+	}
+
+	renderActionTile() {
+		const { isConnected, siteUrl, translate } = this.props;
+		const headerText = translate( 'Keep track of your visitors with Jetpack.' );
+		const subHeaderText = translate(
+			'Keep an eye on your success with simple, concise, and mobile-friendly stats. ' +
+				'Get updates on site traffic, successful posts, site searches, and comments, all in real time.'
+		);
+		const connectUrl = addQueryArgs(
+			{
+				url: siteUrl,
+				// TODO: add a parameter to the JPC to redirect back to this step after completion
+				// and in the redirect URL include the ?action=activate_stats parameter
+				// to actually trigger the stats activation action after getting back to JPO
+			},
+			'/jetpack/connect'
+		);
+		const href = ! isConnected ? connectUrl : null;
+
+		return (
+			<Fragment>
+				<FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />
+
+				<TileGrid>
+					<Tile
+						buttonLabel={ translate( 'Activate stats' ) }
+						image={ '/calypso/images/illustrations/type-business.svg' }
```

Comment:
Double quotes 🙂

Reply:
🤦‍♂️  - addressed in https://github.com/Automattic/wp-calypso/pull/22621/commits/f326b6a9cc84f2dfa7ebb0c50645a50a0e0ae73d.